hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1543,6 +1543,8 @@ message OmDBAccessInfo {
 
 message GetS3VolumeResponse {
   optional VolumeInfo volumeInfo = 1;
+  // Piggybacked user name (principal) response to be used for KMS operations
+  optional string userPrincipal = 2;
```

Comment:
Can we change the name of this proto to `GetS3VolumeInfoResponse`, and change the request name accordingly?  Then they match the name of the RPC call (getS3VolumeInfo) instead of the name of the client side wrapper method (getS3Volume). The current naming makes it seem like getS3Volume is the RPC call, which is a bit confusing if following the flow starting from the client.

I added this proto on our branch, so changing the name now will not introduce any compatibility failures (and the build would fail if it did).

Reply:
Done!